config/initializers/new_framework_defaults.rb

```diff
@@ -21,5 +21,3 @@
 # Require `belongs_to` associations by default. Previous versions had false.
 Rails.application.config.active_record.belongs_to_required_by_default = false
 
-# Do not halt callback chains when a callback returns false. Previous versions had true.
-ActiveSupport.halt_callback_chains_on_return_false = true
```

Comment:
I couldn't find any ActiveRecord or ActiveModel callbacks that were expecting any halting behaviour, so switching to the new behaviour (which unblocks migrating to Rails 5.2 in future) shouldn't cause any issues.